src/shared/lib/Colors.ts

```diff
@@ -83,6 +83,36 @@ export let RESERVED_CLINICAL_VALUE_COLORS: { [value: string]: string } = {
     truncating: MUT_COLOR_TRUNC,
     fusion: MUT_COLOR_FUSION,
     promoter: MUT_COLOR_PROMOTER,
+    // SURVIVAL_DATA: CLI_NO_COLOR
+    '1:deceased': CLI_NO_COLOR,
+    '1:recurred/progressed': CLI_NO_COLOR,
+    '1:recurred': CLI_NO_COLOR,
+    '1:progressed': CLI_NO_COLOR,
+    '1:yes': CLI_NO_COLOR,
+    '1:1': CLI_NO_COLOR,
+    '1:progression': CLI_NO_COLOR,
+    '1:event': CLI_NO_COLOR,
+    '1:dead of melanoma': CLI_NO_COLOR,
+    '1:dead with tumor': CLI_NO_COLOR,
+    '1:metastatic relapse': CLI_NO_COLOR,
+    '1:localized relapse': CLI_NO_COLOR,
+    // SURVIVAL_DATA: CLI_YES_COLOR
+    '0:living': CLI_YES_COLOR,
+    '0:alive': CLI_YES_COLOR,
+    '0:diseasefree': CLI_YES_COLOR,
+    '0:no': CLI_YES_COLOR,
+    '0:0': CLI_YES_COLOR,
+    '0:progressionfree': CLI_YES_COLOR,
+    '0:no progression': CLI_YES_COLOR,
+    '0:not progressed': CLI_YES_COLOR,
+    '0:censored': CLI_YES_COLOR,
+    '0:censor': CLI_YES_COLOR,
+    '0:alive or censored': CLI_YES_COLOR,
+    '0:alive or dead tumor free': CLI_YES_COLOR,
+    '0:no relapse': CLI_YES_COLOR,
+    '0:progression free': CLI_YES_COLOR,
+    '0:censure': CLI_YES_COLOR,
+    '0:ned': CLI_YES_COLOR,
```

Comment:
I think you are missing the space in the middle, that's why the color is not matched.

Reply:
nvm, just saw the message thread in slack. This looks good.

Reply:
wondering if it isn't easier in this case to match on `0:` and `1:`? Guess that would involve more code changes, but kinda feels like one of the reasons for introducing the `0/1` stuff is to normalize the data?